docker/index-gcmd.py

```diff
@@ -0,0 +1,25 @@
+from urllib.request import urlopen
+from xml.dom import minidom
+
+doc = minidom.Document()
+root = doc.createElement("urlset")
+root.setAttribute("xmlns"  , "http://www.sitemaps.org/schemas/sitemap/0.9")
+root.setAttribute("xmlns:xsi", "http://www.w3.org/2001/XMLSchema-instance")
+root.setAttribute("xsi:schemalocation"  , "http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd")
+
+doc.appendChild(root)
+
+for i in range(1,8):
```

Comment:
This works great until more datasets get added and we don't pick them up because they're on page 9!

In order to not have that happen, you'll need to make an initial request first and get the `hits` value to know how many pages there really are, and then keep paging through them until you get to the end.